src/stdx/unit_tests.zig

```diff
@@ -0,0 +1,10 @@
+comptime {
+    _ = @import("stdx.zig");
+    _ = @import("aegis.zig");
+    _ = @import("bit_set.zig");
+    _ = @import("bounded_array.zig");
+    _ = @import("prng.zig");
+    _ = @import("ring_buffer.zig");
+    _ = @import("sort_test.zig");
+    _ = @import("zipfian.zig");
+}
```

Comment:
Rather than adding a separate unit-test file, let's move this to `stdx.zig`, so we use it as a root module for tests as well.